test/powershell/Modules/Microsoft.PowerShell.Utility/Sort-Object.Tests.ps1

```diff
@@ -225,6 +225,80 @@ Describe "Sort-Object DRT Unit Tests" -Tags "CI" {
 	}
 }
 
+Describe 'Sort-Object Stable Unit Tests' -Tags 'CI' {
+
+	Context 'Modulo stable sort' {
+
+		$unsortedData = 1..20
```

Comment:
We could use shorter range  `1..10`.

Reply:
Using a shorter range invalidates the test. A longer range is required for the test to be able to properly identify that the stable sort is working. There is no difference between a stable and an unstable short with a range of 1..10.